apps/song-app/quarkus/song-app/src/main/resources/application.properties

```diff
@@ -1,13 +1,20 @@
+%prod.kafka.bootstrap.servers=kafka:9092
+%dev.kafka.bootstrap.servers=localhost:29092
+%k8s.kafka.bootstrap.servers=my-cluster-kafka-bootstrap:9092
+
 mp.messaging.outgoing.songs.connector=smallrye-kafka
 mp.messaging.outgoing.songs.topic=songs
```

Comment:
Also if channel and topic is named equally I think it can be removed

Reply:
yes -> removed with appended commit